nat-lab/bin/nordlynx

```diff
@@ -9,7 +9,7 @@ ip addr add 100.64.0.1/10 dev nordlynx0  # meshnet
 ip link set mtu 1420 dev nordlynx0
 ip link set dev nordlynx0 up
 
-nlx set nordlynx0 listen-port 51820
```

Comment:
I wanted to complain that this is standard WG port so it's more in-line with Wireguard even though it can be freely chosen, but actually I stand corrected - This port is only used in WG examples.